README.md

```diff
@@ -10,6 +10,20 @@ quality. It uses the dataset generated by the [telemetry-streaming](https://gith
 
 The server-side backend is written in Python using Django. The UI is written in [React](https://reactjs.org/), [Redux](http://redux.js.org/) and [metricsgraphics](http://metricsgraphicsjs.org/).
 
+Getting in touch
+----------------
+
+If you have any questions about Mission Control (either as a user or contributor), the best place to ask is the [#missioncontrol](https://client00.chat.mibbit.com/?server=irc.mozilla.org&channel=%23missioncontrol) channel on irc.mozilla.org ([learn more about irc @ Mozilla](https://wiki.mozilla.org/IRC)).
+
+Contributing
+------------
+
+We welcome contributions to mission control! Working on the UI component (see
```

Comment:
nit: "Mission Control" is capitalized elsewhere, should be consistent.

Reply:
good catch, will fix